src/Stan/Report/Css.hs

```diff
@@ -64,28 +65,75 @@ stanCss = do
     ".exclude" ? color orange
     ".ignore" ? color yellow
 
-    ".centre" ? textAlign center
-
-    -- grid
-    ".container" ? do
-        width (90%)
-        marginLeft auto
-        marginRight auto
-    ".row" ? do
+grid :: Css
+grid = do
+    (html <> body) ? do
+        height (100%)
         width (100%)
-        position relative
-    ".half" ? width (46%)
+        marginAll 0
+        paddingAll 0
+        left (0%)
+        top (0%)
+        fontFamily [] [sansSerif]
+        fontSize (100%)
+        color darkGrey
+        lineHeight (unitless 1.5)
+    ".centre" ? (textAlign center) -- >> marginAuto)
+    ".container" ? (width (90%) >> marginAuto)
+    ".row" ? (position relative >> width (100%))
+    ".row [class^='col']" ? do
+        float floatLeft
+        margin2 (rem 0.5) (2%)
+        minHeight (rem 0.125)
+    sconcat colClasses ? width (96%)
+    colsGrid colClassesSm
+    ".row" # after ? do
+        content (stringContent "")
+        display displayTable
+        clear both
+    ".hidden-sm" ? display displayNone
+    mediaQuery 33.75 $ ".container" ? width (80%)
+    mediaQuery 45 $ colsGrid colClasses
+    mediaQuery 60 $ ".container" ? (width (75%) >> maxWidth (rem 60))
 
   where
-    marginAll :: Size Percentage -> Css
-    marginAll x = margin x x x x
+    cols :: NonEmpty Text
+    cols = fmap ((".col-" <>) . show) $ (1 :: Int) :| [2..12]
+
+    colClasses, colClassesSm :: NonEmpty Selector
+    colClasses   = fmap element cols
+    colClassesSm = fmap (element . (<> "-sm")) cols
+
+    colsGrid :: NonEmpty Selector -> Css
+    colsGrid classes = sequence_ $ NE.map (\(cl, p) -> cl ? width (p %)) $ NE.zip classes w
```

Comment:
Hmm... `map (\(a, b) -> ...) $ zip l r` is `zipWith (\a b -> ...) l r`. And since you want to `sequence_` all actions, I guess you can simply `zipWithM_`:

* https://hackage.haskell.org/package/base-4.14.0.0/docs/Control-Monad.html#v:zipWithM_